README.md

```diff
@@ -156,5 +159,7 @@ or [join our team][join-us], don't hesitate to get in touch.
 [hr-til]: https://github.com/hashrocket/hr-til
 [join-us]: https://hashrocket.com/contact-us/jobs
 [mit]: http://www.opensource.org/licenses/MIT
+[newsletter]: https://www.getrevue.co/profile/til/
 [oauth_google]: https://developers.google.com/identity/protocols/OAuth2WebServer
+[twitter]: https://twitter.com/hashrockettil/
```

Comment:
Both URLs still work with the trailing `/`, so no issue, but is there a reason to have them? I'm assuming you copy/pasted it from the browser so I was just curious how they got there? Traditionally that meant the URL was a folder, but I don't think thats the case in modern web.

Reply:
I don't have a reason to use the trailing slashes; I append them unconsciously. This SO answer sums up some different perspectives:

https://stackoverflow.com/a/5949201/2112512

I guess I'm in the "makes the address look pretty" camp. I'll take them out.

Reply:
no worries :) . Thought it might be some cool SEO or A11y hack or something. Was mostly curious. 👍